runbotics/runbotics-orchestrator-ui/src/src-app/components/Tile/ProcessTile/ProcessTileTag/ProcessTileTag.styles.ts

```diff
@@ -0,0 +1,88 @@
+import ExpandMoreIcon from '@mui/icons-material/ExpandMore';
+import { Typography } from '@mui/material';
+import styled from 'styled-components';
+
+export const StyledTypography = styled(Typography)(({ theme }) => `
+    && {
+        color: ${theme.palette.grey[500]};
+        text-transform: lowercase;
+        position: relative;
+        top: 7px;
+        user-select: none;
+
+        &:hover {
+            cursor: pointer;
+        }
+    }
+`);
+
+export const StyledExpandIcon = styled(ExpandMoreIcon)<{ $isExpanded: boolean; }>`
+    && {
+        position: relative;
+        top: 10px;
+        color: ${props => props.theme.palette.grey[500]};
+        transform: rotate(${props => props.$isExpanded ? '-180deg' : '0'});
+        transition: 0.5s;
+    }
+`;
+
+export const Container = styled.div`
+    display: flex;
+    flex-wrap: wrap;
+    position: relative;
+    top: -8px;
+    height: 20px;
+`;
+
+export const TagBox = styled.div<{ $isExpanded: boolean; }>`
+    display: flex;
+    flex-wrap: wrap;
+    align-content: flex-start;
+    height: ${props => props.$isExpanded ? '170px' : '130%'};
```

Comment:
why the $ sign?, imo redundant

Reply:
I need more info about those values `170px` and `130%`

Reply:
$ is from official styled-components documentation: https://styled-components.com/docs/basics#passed-props

Reply:
170px and 130% - I have to adjust those values to process tile component. 170px is when TagBox is not expanded, otherwise 130%

Reply:
> $ is from official styled-components documentation: https://styled-components.com/docs/basics#passed-props

yup, without the $ the HTML yells there cannot be custom props passed to div element (styled-component)